test/e2e/dra/deploy.go

```diff
@@ -84,10 +88,56 @@ func NewNodes(f *framework.Framework, minNodes, maxNodes int) *Nodes {
 			nodes.NodeNames = append(nodes.NodeNames, node.Name)
 		}
 		framework.Logf("testing on nodes %v", nodes.NodeNames)
+
+		// Watch claims in the namespace. This is useful for monitoring a test
+		// and enables additional sanity checks.
+		claimInformer := resourceapiinformer.NewResourceClaimInformer(f.ClientSet, f.Namespace.Name, 100*time.Hour /* resync */, nil)
+		cancelCtx, cancel := context.WithCancelCause(context.Background())
```

Comment:
use a ktesting context?

Reply:
I want to do a full pass through this file and update it to use ktesting a bit more, but not now.